torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1139,7 +1139,10 @@ Tensor cholesky_jvp(const Tensor& input_tangent, const Tensor& L, bool upper) {
   auto input_tangent_ = upper ? input_tangent.mH() : input_tangent;
   auto L_ = upper ? L.mH() : L;
 
-  auto L_inverse = at::linalg_solve_triangular(L_, at::eye(L.size(-1), L.options()), /*upper=*/false);
```

Comment:
Again, it's fine to keep this but it should also continue working without this change.